src/pages/404.jsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import { Result, Button, Typography } from 'antd';
-
-import PropTypes from 'prop-types';
 import FeedbackButton from '../components/FeedbackButton';
 
-const { Title, Paragraph } = Typography;
```

Comment:
doesn't look like `Paragraph` is used, so you can delete it

Reply:
It is deleted, again it is a negative diff.